src/pepr/operator/controllers/reload/reload-utils.ts

```diff
@@ -22,8 +22,15 @@ import { createEvent, retryWithDelay } from "../utils";
  * @param pods List of pods to evict or restart
  * @param reason The reason for eviction/restart (for logging)
  * @param log Logger instance for logging
+ * @param resource Resource type responsible for eviction/restart (for logging)
  */
-export async function reloadPods(namespace: string, pods: kind.Pod[], reason: string, log: Logger) {
+export async function reloadPods(
+  namespace: string,
+  pods: kind.Pod[],
+  reason: string,
+  log: Logger,
+  resource: string,
```

Comment:
If we do want to keep this parameter around (see above comment) it should be called `resourceType` and use the appropriate typing.  But this makes this function specific to only being used by ConfigMap and Secret reloads.

```suggestion
  resourceType: ResourceType,
```